apache/app.mako-dot-conf

```diff
@@ -56,6 +57,15 @@ RewriteRule ${apache_base_path}/([0-9a-f]+)/(services|layersConfig)  ${apache_ba
    Allow from all
 </LocationMatch>
 
+% if apache_base_path == '':
+# Serve everything but personal directory and configs from S3 bucket
+RewriteCond %{REQUEST_URI} !^/lt[a-z]{3}
+RewriteCond %{REQUEST_URI} !^/configs
+RewriteRule ^/(.*) http://s3-eu-west-1.amazonaws.com/${s3_bucket_dev}/$1 [P]
```

Comment:
does this work or should it be `http://${s3_bucket_dev}.s3-eu-west-1.amazonaws.com/$1`

Reply:
Both are OK (tested, sorry)

Reply:
Served by apache https://mf-geoadmin3.dev.bgdi.ch/ltmom/ (as /ltboc, etc..)
Not served by apache https://mf-geoadmin3.dev.bgdi.ch/index.html (only to test)